development/gcp/cloud-functions/getGithubIssue/getgithubissue.go

```diff
@@ -223,30 +252,34 @@ func GetGithubIssue(ctx context.Context, m kymapubsub.MessagePayload) error {
 			}
 			logger.LogInfo(fmt.Sprintf("published pubsub message to topic %s, id: %s", getFailureInstanceTopic, *publlishedMessageID))
 		}
-		// When failing tests message has no github issue number.
+		// Failing tests message has no github issue number.
 	} else {
 		// Create github issue.
-		ghIssue, err := createGithubIssue(ctx, githubClient, failingTestMessage, githubOrg, githubRepo)
+		ghIssue, err := createGithubIssue(ctx, githubClient, failingTestMessage)
 		if err != nil {
 			logger.LogError(fmt.Sprintf("failed create github issue, error: %s", err.Error()))
 		}
 		if ghIssue != nil {
 			logger.LogInfo(fmt.Sprintf("github issue created. issue number: %d", ghIssue.GetNumber()))
-			if ghIssue.Number != nil {
-				// Add github issue number to failing test pubsub message.
-				failingTestMessage.GithubIssueNumber = github.Int64(int64(*ghIssue.Number))
-				docRef := firestoreClient.Doc(fmt.Sprintf("%s/%s", firestoreCollection, *failingTestMessage.FirestoreDocumentID))
-				// Add created github issue number to failing test instance document in firestore.
-				_, err = docRef.Set(ctx, map[string]int{"githubIssueNumber": *ghIssue.Number}, firestore.Merge([]string{"githubIssueNumber"}))
-				if err != nil {
-					logger.LogError(fmt.Sprintf("failed adding github issue number %d, to failing test instance, error: %s", ghIssue.GetNumber(), err.Error()))
-					// TODO: need error reporting for such case, without failing whole function
-				} else {
-					logger.LogInfo(fmt.Sprintf("github issue, number %d, added to failing test instance", ghIssue.GetNumber()))
-				}
-			} else {
-				logger.LogError(fmt.Sprintf("github issue number is nil, something went wrong with creating github issue"))
+			// Add github issue number to failing test pubsub message.
+			failingTestMessage.GithubIssueNumber = github.Int64(int64(*ghIssue.Number))
```

Comment:
Can `ghIssue.GetNumber()` be used here?

Reply:
No, because GetNumber return int, while I need a *int.